packages/katello/katello/katello-service

```diff
@@ -78,9 +77,6 @@ def manage_services(action)
       when 'enable', 'disable'
         puts "systemctl #{action} #{service}.service"
         puts `systemctl #{action} #{service}.service`
-      else
```

Comment:
This doesn't sound right. I you're breaking all actions besides `enable` and `disable`. The proper way is to map everything to systemctl (so remove the case statement and just include the 2 `puts` lines).